pkg/datapath/linux/config/config.go

```diff
@@ -105,6 +111,18 @@ func (h *HeaderfileWriter) WriteNodeConfig(w io.Writer, cfg *datapath.LocalNodeC
 			ipv4NP := node.GetNodePortIPv4()
 			cDefinesMap["IPV4_NODEPORT"] = fmt.Sprintf("%#x", byteorder.HostSliceToNetwork(ipv4NP, reflect.Uint32).(uint32))
 		}
+
+		if option.Config.EnableIPv4FragmentsTracking {
+			pm := probes.NewProbeManager()
+			supportedMapTypes := pm.GetMapTypes()
+			if supportedMapTypes.HaveLruHashMapType {
+				cDefinesMap["IPV4_FRAGMENTS"] = "1"
+				cDefinesMap["IPV4_FRAG_DATAGRAMS_MAP"] = fragmap.MapName
+				cDefinesMap["CILIUM_IPV4_FRAG_MAP_MAX_ENTRIES"] = fmt.Sprintf("%d", fragmap.MaxEntries)
+			} else {
+				log.Warningf("No support for IPv4 fragments (LRU maps not supported by kernel)")
```

Comment:
Do we need this to be `Warning` or should it just be `Info`? People follow metrics / set alerts on the number of warnings we report from the daemon. Also, the `XXXf` form is not necessary here as there are no additional arguments. We typically try to avoid this form in favour of `log.WithFields(...).Info()`.

```suggestion
				log.Info("No support for IPv4 fragments (LRU maps not supported by kernel)")
```

Reply:
I already noticed and replaced the `XXXf` form with `Warning()`. But I don't think we _need_ a warning here, so I'll just go with `Info()`.